server/src/test/java/org/opensearch/index/remote/RemoteStoreUtilsTests.java

```diff
@@ -151,10 +151,10 @@ public void testVerifyMultipleWriters_Segment() {
     }
 
     public void testVerifyMultipleWriters_Translog() throws InterruptedException {
-        TranslogTransferMetadata tm = new TranslogTransferMetadata(1, 1, 1, 2, "node-1");
+        TranslogTransferMetadata tm = new TranslogTransferMetadata(1, 1, 1, 2, "node--1");
```

Comment:
curious on why did we have to add one more hyphen?

Reply:
This is just to test that if we have a separator string `--` inside node name, it doesn't break our comparison.